src/lite-yt-embed.css

```diff
@@ -83,3 +83,36 @@ lite-youtube.lyt-activated > .lty-playbtn {
     white-space: nowrap;
     width: 1px;
   }
+
+lite-youtube .lyt-title-text {
+    font-family: "YouTube Noto",Roboto,Arial,Helvetica,sans-serif;
+    color: hsla(0,0%,100%,.9);
+    text-align: left;
+    direction: ltr;
```

Comment:
i think this is a bit overstyled.  i want the CSS & JS to be as small as reasonably possible, so i'd like to only include these if totally necessary.  Can you whittle these down a bit?

Reply:
Removed extraneous things, and moved styles to ::before element. Also, I changed the background from a base64 encoded svg into a linear-gradient, because the svg isn't stretching correctly when title text expands the element.